conans/client/generators/pkg_config.py

```diff
@@ -61,14 +61,18 @@ def single_pc_file_contents(self, name, cpp_info):
         lines.append("Version: %s" % cpp_info.version)
         libdirs_flags = ["-L${%s}" % name for name in libdir_vars]
         libnames_flags = ["-l%s " % name for name in cpp_info.libs]
+        frameworks = format_frameworks(cpp_info.frameworks, compiler=self.compiler)
```

Comment:
Same comment, should we double-check this for Macos?

Reply:
added sanity check